spec/models/participant_profile/ecf_spec.rb

```diff
@@ -210,4 +210,22 @@
       end
     end
   end
+
+  describe "#role" do
+    context "when participant is Mentor" do
+      let(:profile) { create(:mentor_participant_profile) }
```

Comment:
i probably would have chosen to use `build` here to do it in memory rather than make the db call

Reply:
I went with the `create` because the `build(:mentor_participant_profile)` is failing